climpred/checks.py

```diff
@@ -180,3 +182,58 @@ def has_valid_lead_units(xobj):
     else:
         raise AttributeError(LEAD_UNIT_ERROR)
     return True
+
+
+NCPU = multiprocessing.cpu_count()
+
+
+def get_chunksize(da):
```

Comment:
Can you insert this alphabetically into the existing module? I've been trying to make a practice of that in `utils` and `checks` for more clear organizing.